src/lib.rs

```diff
@@ -286,6 +286,9 @@ cfg_if! {
     } else if #[cfg(target_os = "redox")] {
         mod redox;
         pub use redox::*;
+    } else if #[cfg(target_os = "fuchisa")] {
```

Comment:
There's a typo: `fuchisa`, so this is likely not going to work.

Reply:
Gah! https://github.com/rust-lang/libc/pull/854